design/clusters.go

```diff
@@ -168,6 +168,22 @@ var _ = a.Resource("clusters", func() {
 		a.Response(d.InternalServerError, JSONAPIErrors)
 	})
 
+	a.Action("delete", func() {
+		a.Security("jwt")
+		a.Routing(
+			a.DELETE("/:clusterID"),
+		)
+		a.Params(func() {
+			a.Param("clusterID", d.UUID, "the ID of the cluster to delete")
+			a.Required("clusterID")
+		})
+		a.Description("Delete a cluster configuration")
+		a.Response(d.NoContent)
+		a.Response(d.BadRequest, JSONAPIErrors)
```

Comment:
There can't be any BadRequiest responses, right?

Reply:
right. Removed in 870041f